File: grpc-server/src/main/java/ru/romanow/protocols/grpc/GrpcServerApplication.java
package ru.romanow.protocols.grpc;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * Created by romanow on 03.10.17.
 */
@SpringBootApplication
public class GrpcServerApplication {
    public static void main(String[] args) {
        SpringApplication.run(GrpcServerApplication.class, args);
    }
}

File: grpc-server/src/main/java/ru/romanow/protocols/grpc/GrpcServerApplication.kt
package ru.romanow.protocols.grpc

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class GrpcServerApplication

fun main(args: Array<String>) {
    SpringApplication.run(GrpcServerApplication::class.java, *args)
}
